readme.txt

```diff
@@ -4,7 +4,7 @@ Donate link:       https://10up.com/plugins/restricted-site-access-wordpress/
 Tags:              privacy, restricted, restrict, privacy, limited, permissions, security, block
 Requires at least: 5.7
 Tested up to:      6.0
```

Comment:
Do you want to give this a quick test on 6.1 RC4 and bump this to 6.1 if things test as expected?

Reply:
Tested with 6.1-RC5, all tests pass 👍